### ISSUE TITLE
[number of seconds to group by] 

### ISSUE THREAD

**Issue by YYYYWD** [creotiv/object_detection_projects#7]
I don't know what is the mean of " [number of seconds to group by] " 

how can i do if i want to python plot.py ?

**Comment by creotiv** [comment]
that's my bad english) Should be [number of seconds to aggregate by]
It's just saying width in seconds of window which will be aggregated in one value.

**Comment by YYYYWD** [comment]
OK fine.
But I find some problems.
This is your the past code
![image](https://user-images.githubusercontent.com/25359077/38628955-41c30c6e-3de5-11e8-8ef0-254c5916cb6e.png)
I don‘t know how to find these 'car' and 'bus'. Because your the test video of 'report.csv' just has 
![h1 r4 q 0cel i5d fs 7va](https://user-images.githubusercontent.com/25359077/38629149-d960253e-3de5-11e8-9ad4-1225d034a4aa.png)
So I have to change the code 
![0fd l5qa6 v2h6 e8mx7m](https://user-images.githubusercontent.com/25359077/38629190-f490ae64-3de5-11e8-8358-191a9e2471da.png)
Could you tell me how I should do that?

-----------------------------
And If I change the Green shade's position ranges.What should I do?

**Comment by creotiv** [comment]
yes, because stats has only vehicles and not type classification, you need to change plot to use stats that you have. You did it.

Green area is an exit zone, where cars counting happens. To change them, you need to change
EXIT_PTS variable in the traffic.py. it just (x,y) points that are creating poligon(it can be more then 4 to create any figure you need.)

**Comment by creotiv** [comment]
Better use pretrained models first like YOLO3 or SSD, if they work, then just use them.

**Comment by YYYYWD** [comment]
Yes，my data is from the YOLO. And I have an idea. I want to judge the abnormal behavior of the vehicle.I want to base on the trajectory of the vehicle.If I can use your code's 'path' to do that.Or do you have any better Suggestions?

**Comment by YYYYWD** [comment]
I have a new question. I used my own test video, which changed the export area. But it was found that only two cars could be detected at any one time. If there are three cars left, they can't be detected. What do I need to change?

**Comment by zubairahmed-ai** [comment]
Facing the same issue 
`KeyError: 'car'`

If you are recommending to use your own video then please also change the code to remove this key and use the generic 'vehicles' key as you suggested above
